lld/docs/ReleaseNotes.rst

```diff
@@ -29,6 +29,9 @@ ELF Improvements
 * ``--compress-sections <section-glib>=[none|zlib|zstd]`` is added to compress
   matched output sections without the ``SHF_ALLOC`` flag.
   (`#84855 <https://github.com/llvm/llvm-project/pull/84855>`_)
+* ``--compression-level`` is added to set the compression level
+  for ``--compress-debug-sections=[zlib|zstd]`` and ``--compress-sections``.
+  (`#90567 <https://github.com/llvm/llvm-project/pull/90567>`_)
```

Comment:
Worth mentioning that default compression level is now independent of linker optimization level (Z_BEST_SPEED)?

Reply:
thx for the suggestion. adopting